### PR TITLE
[Needs Review] r06522601 蔡誌烜

### PR DESCRIPTION
**Pull request by ZHsuan**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## What is the most challenging part in this homework?
I can't pass task8
## Is this homework helpful for you?
Yes
## Is there any suggestion?

<br />

### CODE REVIEW

**Comment by ZHsuan** [comment]
It shows "Review required" does it mean my PR failed? Could I still fix it now or it's too late? Thanks. @amjltc295

**Comment by amjltc295** [comment]
@ZHsuan  It simply means I have not reviewed your code. Just wait for a while.

File: src/students/r06522601.py [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    Returns:
+        student: an Student object
+
+    Hints:
+        * Use Student(parameters1, parameters2 ...) to create an object
+          and assign it to a variable
+        * Use <created object>.<object function> to call object function
+    '''
+    # TODO: create a student object with different words to say
+    student = Student(student_id, time)
```
Comment (amjltc295) [inline src/students/r06522601.py]: Did you test this task?

File: src/students/r06522601.py [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+
+    # Display the image:
+    # result_img.show()
+    # Note: please comment this line when hand in.
+
+    # If you are running on a server, use
+    # result.save('test.jpg')
+    # and copy the file to local or use Jupyter Notebook to render.
+    reslt_img = request.urlopen(img_url)
+    from .. import ultis
```
Comment (amjltc295) [inline src/students/r06522601.py]: What is this? Did you test this task?

File: src/students/r06522601.py (hunk 1 above)
Comment (ZHsuan) [reply src/students/r06522601.py]: Yes I did this one

File: src/students/r06522601.py (hunk 2 above)
Comment (ZHsuan) [reply src/students/r06522601.py]: It's wrong because I didn't find the correct answer for task 8

**Pull request merged**